ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,179 @@
+# Alzadaal's Legacy
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# SILKIE #
+#~~~~~~~~#
+
+# -ii 77ED 7762 7768 7769 776A
+
+0.0 "--sync--" sync / 00:0839::The Trial of Knowledge is sealed off/ window 0,1
+12.2 "--sync--" sync / 14:[^:]*:Silkie:7759:/ window 15,15
+14.9 "Fizzling Suds" sync / 1[56]:[^:]*:Silkie:7759:/
+21.0 "Soap's Up" sync / 1[56]:[^:]*:Silkie:775A:/
+22.0 "Fizzling Duster" sync / 1[56]:[^:]*:Silkie:775D:/
+32.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+38.4 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+43.5 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+50.1 "Squeaky Clean" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+51.6 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+53.3 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+54.8 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+61.9 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+64.0 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+69.0 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+69.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:7760:/
+73.0 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+80.6 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+89.7 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+89.7 "Chilling Duster and Bracing Duster" #sync / 1[56]:[^:]*:Silkie:(7768|7769):/
+100.9 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+106.0 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+112.1 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+123.1 "Puff and Tumble" duration 0.7 #sync / 1[56]:[^:]*:Silken Puff:7771:/
+129.5 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+129.5 "Fizzling Duster x2 and Chilling Duster x2" #sync / 1[56]:[^:]*:Silken Puff:(776A|7768):/
```

Comment:
Is this Fizzling or Chilling? Or both at the same time?

If either, please say: `Fizzling Duster x2/Chilling Duster x2` and add an English replacement in `timelineReplace` to make it say `Fizzling/Chilling Duster x2` in English.

If both, please put them on separate lines.

EDIT: same comments for the below "and" duster lines.

Reply:
The one here is always x2 of both. The others can vary. I'll have to review what's possible at each...
Soaping Spree 1 (Slippery Soap hits non Chilling Duster):
- Chilling Duster + Fizzling Duster + Bracing Duster
- Chilling Duster x2 + Bracing Duster
Soaping Spree 1 (Slippery Soap misses dusters):
- Bracing Duster x2 + Chilling Duster
- Fizzling Duster x2 + Bracing Duster
- Bracing Duster x2 + Fizzling Duster

Soaping Spree 2:
- 2x Chilling Duster + 2x Fizzling Duster + 1x Bracing Duster (boss)
- 2x Chillind Duster + 2x Fizzling Duster + 1x Chillind Duster (boss)

Soaping Spree 3:
- 2x Fizzling Duster
- 1x Bracing Duster + 1x Fizzling Duster
- 1x Fizzling Duster + 1x Chilling Duster

Soaping Spree 4:
- 0x - 6x Chilling Duster + Bracing Duster (boss)

Soaping Spree 5:
- 4x Chilling Duster +  1x Bracing Duster (boss)

Soaping Spree 5 (if missed any puffs during Slippery soap)
- 0-6x Bracing Duster + 4x Chilling Buster + 1x Bracing Duster (boss)
- 
Soaping Spree 5 (if faced tether wrong way)
- 0-4x Chilling Duster + 0-4x Chilling Duster + 1x Bracing Duster (boss)

Soaping Spree 5 (Failing multiple soaping spree checks):
- 0-7x Bracing Duster + 0-8x Chilling Duster + 1x  Bracing Duster (boss)

I think you can get somewhat lucky and have puffs be on safe side and become Chillind Dusters as well, but I think this last Spree just cascades the explosions. Also, you can have limit break 2 from tank here and survive a bad duster or two.

These can also trigger chain reactions if mechanics are done wrongly. I've seen some weird interactions where dusters go off before Soaping Spree finishes cast.